src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/Features/ApprenticeshipIncentiveCreated.feature

```diff
@@ -24,4 +24,9 @@ Scenario: Apprenticeship incentive earnings are calculated
 Scenario: Incentive application updated after earnings calculated
 	Given an apprenticeship incentive earnings have been calculated
 	When the earnings calculation against the apprenticeship incentive completes
-	Then the incentive application is updated to record that the earnings have been calculated
\ No newline at end of file
+	Then the incentive application is updated to record that the earnings have been calculated
+
+Scenario: Apprenticeship incentive is withdrawn
+	Given an apprenticeship incentive exists
+	When the apprenticeship incentive is withdrawn
```

Comment:
The When is what happens so should be the same as the others.  So "When the earnings calculation against the apprenticeship completes".  The Given is the starting point so is that the apprenticeship is withdrawn .